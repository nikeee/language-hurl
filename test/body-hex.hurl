PUT https://example.org
# Send a caf√©, encoded in UTF-8
hex,636166c3a90a;
