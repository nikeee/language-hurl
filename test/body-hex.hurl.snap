>PUT https://example.org
#^^^ source.hurl keyword.control.method.hurl
#   ^ source.hurl
#    ^^^^^^^^^^^^^^^^^^^ source.hurl string.quoted.url.hurl string.quoted.url.hurl
># Send a café, encoded in UTF-8
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl comment.line.number-sign.hurl
>hex,636166c3a90a;
#^^^ source.hurl support.function support.function.name
#   ^ source.hurl support.function
#    ^^^^^^^^^^^^ source.hurl support.function text.plain
#                ^ source.hurl support.function support.function
>